framework/SupervisedLearning/SciKitLearn.py

```diff
@@ -177,9 +177,11 @@ def __init__(self,messageHandler,**kwargs):
     """
     supervisedLearning.__init__(self,messageHandler,**kwargs)
     name  = self.initOptionDict.pop('name','')
+    # some keywords aren't useful for this ROM
     if 'pivotParameter' in self.initOptionDict:
```

Comment:
This can be removed.

Reply:
Why would we remove this comment when it explains the actions of the lines below the comment?

Reply:
I mean you do not need the if statement